$NetBSD: patch-tools_libs_ctrl_Makefile,v 1.1 2021/04/18 12:31:26 bouyer Exp $

PKG_CONFIG is an environnement variable, rename to PKG_CONFIG_PC

--- tools/libs/ctrl/Makefile.orig	2020-12-09 18:39:19.902854035 +0100
+++ tools/libs/ctrl/Makefile	2020-12-09 18:39:37.171930411 +0100
@@ -50,7 +50,7 @@
 CFLAGS-$(CONFIG_Linux) += -D_GNU_SOURCE
 
 LIBHEADER := xenctrl.h xenctrl_compat.h
-PKG_CONFIG := xencontrol.pc
+PKG_CONFIG_PC := xencontrol.pc
 PKG_CONFIG_NAME := Xencontrol
 
 NO_HEADERS_CHK := y
